crates/re_data_loader/src/load_file.rs

```diff
@@ -113,6 +113,10 @@ pub(crate) fn prepare_store_info(
                 is_official_example: false,
                 started: re_log_types::Time::now(),
                 store_source,
+                // NOTE: Since this cannot be an RRD file, either the current process or an external data
+                // loader will have to do the logging.
```

Comment:
assuming that external data loaders are up-to-date seems weird. Can't we just use `None` here instead?

Reply:
Probably -- I'll try it out

Reply:
That made me realize we have all the information we need.

We now force `LOCAL` for builtin loaders, and let external loaders set it otherwise.